app/containers/Facilities/FacilitiesListContainer.tsx

```diff
@@ -53,9 +53,9 @@ export const FacilitiesList: React.FunctionComponent<Props> = ({
   const displayNameToColumnNameMap = {
     'Organisation Name': 'organisation_name',
     'Facility Name': 'facility_name',
-    Address: 'facility_mailing_address',
-    'Postal Code': 'facility_postal_code',
-    City: 'facility_city',
+    'Facility Type': 'facility_type',
+    'BC GHG id': 'facility_bcghgid',
+    'Reporting period of last SWRS report': 'reporting_period_duration',
```

Comment:
It looks like reporting_period_duration has been named differently here, in search_all_facilties.sql, and the type facility_search_result. (reporting_period_duration in the search function & last_swrs_reporting_year in the search result type)